src/redux/books/books.js

```diff
@@ -0,0 +1,20 @@
+const ADDBOOK = {
+  type: 'ADD',
+};
+
+const REMOVEBOOK = {
+  type: 'REMOVEBOOK',
+};
+
```

Comment:
- [ ] Kindly refer to comment ... for the `REMOVEBOOK` action creator

#### Hint: The `payload` for the `REMOVEBOOK` action creator should be the `id` of the book that will be filtered out when `REMOVEBOOK` action is triggered 💻